mk

```diff
@@ -51,13 +52,23 @@ case $1 in
                 docker volume rm $(docker volume ls -q);
                 ;;
 
-            "reload")
+            "remake")
                 shift 2
                 ./mk docker rmall;
                 ./mk docker up --force-recreate --build $@;
                 ;;
+            
+            "restart")
+                docker container ls --format="{{.Names}}" | fzf --preview="docker inspect {1}" --bind "enter:become(docker container restart {1})"
```

Comment:
Using fzf here is a bad idea
I need `mk` to be without dependencies that are external to the project

Reply:
Issue Opened [here](https://github.com/samuel-joly/Montcelard-event/issues/20)